packages/server/modules/notifications/helpers/types.ts

```diff
@@ -66,6 +70,29 @@ export type MentionedInCommentMessage = NotificationMessage<
   MentionedInCommentData
 >
 
+export type NewStreamAccessRequestData = { requestId: string }
+
+export type NewStreamAccessRequestMessage = NotificationMessage<
+  NotificationType.NewStreamAccessRequest,
+  NewStreamAccessRequestData
+>
+
+export type StreamAccessRequestApprovedData = {
```

Comment:
odd that it counts these stream access request changes as yours 🤔 some kind of merge issue maybe?